client/src/components/LanguageOptions.js

```diff
@@ -1,25 +1,13 @@
 const languageOptions = [
-  { key: "Ar", text: "Arabic", value: "Arabic" },
-  { key: "Chi", text: "Chinese", value: "Chinese" },
-  { key: "Dan", text: "Danish", value: "Danish" },
-  { key: "Dut", text: "Dutch", value: "Dutch" },
-  { key: "En", text: "English", value: "English" },
-  { key: "Fre", text: "French", value: "French" },
-  { key: "Ger", text: "German", value: "German" },
-  { key: "Grek", text: "Greek", value: "Greek" },
-  { key: "Hun", text: "Hungarian", value: "Hungarian" },
-  { key: "Ita", text: "Italian", value: "Italian" },
-  { key: "Jap", text: "Japanese", value: "Japanese" },
-  { key: "Kor", text: "Korean", value: "Korean" },
-  { key: "Lith", text: "Lithuanian", value: "Lithuanian" },
-  { key: "Per", text: "Persian", value: "Persian" },
-  { key: "Pol", text: "Polish", value: "Polish" },
-  { key: "Por", text: "Portuguese", value: "Portuguese" },
-  { key: "Rus", text: "Russian", value: "Russian" },
-  { key: "Spa", text: "Spanish", value: "Spanish" },
-  { key: "Swe", text: "Swedish", value: "Swedish" },
-  { key: "Tur", text: "Turkish", value: "Turkish" },
-  { key: "Viet", text: "Vietnamese", value: "Vietnamese" }
+  { key: "zh-cn", text: " MandarinChinese", value: "Mandarin Chinese" },
```

Comment:
You don't need the text in every language, with the key and the value is enough

Reply:
it is not working when i remove the text in the front end

Reply:
you are right, we need text and value. But the value should contain what we call the 'language code' eg: fr --> for French,  es --> for spanish